### PR TITLE
12 generate main app and landing

### LINKED ISSUE #12: generate main, app and landing
Install react router dom and autoprefixer
Also create the landing and home files
Modify app and main styles

### PR DESCRIPTION
**Pull request by jesusre890**

### CODE REVIEW

File: client/src/Views/Home/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+
+
```
Comment (tamaraantonella) [inline client/src/Views/Home/Home.tsx]: Please remove this extra spaces.

File: client/src/Views/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const Home = () => {
+  return (
+    <div>
+
```
Comment (tamaraantonella) [inline client/src/Views/Home/Home.tsx]: Same as previous comment.

**Review by tamaraantonella — CHANGES_REQUESTED** [review]

File: client/src/Views/Home/Home.tsx (hunk 1 above)
Comment (jesusre890) [reply client/src/Views/Home/Home.tsx]: i am ready

File: client/src/Views/Home/Home.tsx (hunk 2 above)
Comment (jesusre890) [reply client/src/Views/Home/Home.tsx]: I already resolved it

**Review by tamaraantonella — APPROVED** [review]

**Pull request merged**